packages/daemon/src/app/controller/job.ts

```diff
@@ -4,6 +4,8 @@ import { PipelineService } from '../../service/pipeline';
 import { parseConfig } from '../../runner/helper';
 import ServerSentEmitter from '../../utils/emitter';
 import { JobModel } from '../../model/job';
+import { PluginManager } from '../../service/plugin';
+import { constants } from '@pipcook/pipcook-core';
```

Comment:
Name resolving should be put behind the relative import.

Reply:
Done.